benchmarks/performance-poetry/complex-benchmark/src/main/java/com/squareup/benchmarks/performance/poetry/complex/benchmark/ComplexPoetryResults.txt

```diff
@@ -0,0 +1,106 @@
+
+April 13
+Nexus 6
+
+Commit SHA - 7fe2fe3cbe2ef6908ce60a7876122089aaae8503
+
+--------- Workflow Renderings Cumulative Stats ---------
+Total Render Passes So Far: 32
+Total Fresh Rendering Ratio: 0.14360313315926893
+
+Starting 1 tests on Pixel 6 - 12
+Connected to process 5369 on device 'google-pixel_6-1C041FDF60017G'.
+
+ComplexPoetryBenchmarks_benchmarkTraceSectionsFullAOT
+RenderPass-10Ms   min 0.2,   median 0.3,   max 0.5
+RenderPass-11Ms   min 0.5,   median 0.7,   max 0.8
+RenderPass-12Ms   min 0.1,   median 0.3,   max 0.7
+RenderPass-13Ms   min 0.4,   median 0.5,   max 0.7
+RenderPass-14Ms   min 0.2,   median 0.2,   max 0.4
+RenderPass-15Ms   min 0.3,   median 0.7,   max 0.8
+RenderPass-16Ms   min 0.1,   median 0.3,   max 0.9
+RenderPass-17Ms   min 0.4,   median 0.5,   max 0.6
+RenderPass-18Ms   min 0.2,   median 0.3,   max 0.4
+RenderPass-19Ms   min 0.2,   median 0.8,   max 1.0
+RenderPass-1Ms   min 0.2,   median 0.2,   max 0.5
+RenderPass-20Ms   min 0.3,   median 0.4,   max 0.9
+RenderPass-21Ms   min 0.3,   median 0.5,   max 0.6
+RenderPass-22Ms   min 0.2,   median 0.3,   max 0.4
+RenderPass-23Ms   min 0.2,   median 0.5,   max 0.9
+RenderPass-24Ms   min 0.1,   median 0.3,   max 0.6
+RenderPass-25Ms   min 0.3,   median 0.5,   max 0.6
+RenderPass-26Ms   min 0.2,   median 0.3,   max 0.4
+RenderPass-27Ms   min 0.7,   median 0.8,   max 0.9
+RenderPass-28Ms   min 0.3,   median 0.4,   max 0.6
+RenderPass-29Ms   min 0.4,   median 0.5,   max 1.3
+RenderPass-2Ms   min 0.0,   median 0.0,   max 0.2
+RenderPass-30Ms   min 0.2,   median 0.3,   max 0.5
+RenderPass-31Ms   min 0.4,   median 0.8,   max 1.0
+RenderPass-3Ms   min 0.0,   median 0.1,   max 0.2
+RenderPass-4Ms   min 0.3,   median 0.9,   max 1.1
+RenderPass-5Ms   min 0.1,   median 0.3,   max 0.5
+RenderPass-6Ms   min 0.1,   median 0.2,   max 0.4
+RenderPass-7Ms   min 0.2,   median 0.6,   max 0.7
+RenderPass-8Ms   min 0.1,   median 0.2,   max 0.4
+RenderPass-9Ms   min 0.4,   median 0.6,   max 1.3
+RenderPassTotalMs   min 0.2,   median 0.2,   max 0.8
```

Comment:
This is the average of averages. The average of each of 32 render pass durations for each of 20 test iterations.

Reply:
I'm confused, what exactly is the average of average is all these values? The middle value says median.

Also you might want to sort these properly from 1 to 30 ?

I got confused what  `-30Ms` might mean and then just realized it's likely the unit, ms? `ms` should always be written lowercase and with a space, although I get that here its more like a class name. I'd suggest changing to `RenderPass 9 (ms)`